docs/CLI/gyro_offset_yaw.md

```diff
@@ -0,0 +1,35 @@
+# CLI parameter <gyro_offset_yaw>
+
+## Introduction
+This parameter allows to manually compensate YAW gyro drift over time.
+
+## Audience
+This parameter is intended for LOS flyers who use the <HEADFREE> feature of betaflight.
+It reduces the usage of on sight realignment <HEADADJ>
+form 3-6 times to 0-1 during a flight.
+
+## Warnings
+Users with exotic board alignment configurations (this feature may not work as intended for your purpose !!!)
+The gyro drift depends on many factors, so the results of manual calibration have a big variance
+Factors:
+  gyro (mpu6000 ICM260X ...)
+  sampling rate
+  temperature
+  LiPo (3S ... 5S)
+  Flight style
+  Flight time
+  and many more ...
+
+## Setup
+The basic setup can be done on the bench without LiPo (USB powerd)
+power on, do not move the quad for 10 minutes
+on CW  drift 70° --> gyro_offset_yaw = 70
```

Comment:
It seems that gyro_offset_yaw = 1 is 3.7 deg per 10min
http://www.wolframalpha.com/input/?i=(2000%2F2%5E15)%2F100+(angular+degree)%2Fs+*+10+minutes

Reply:
@ledvinap 
I do not know If that is a good measurement point.
I had about 70 -120°/10 minutes flight
and my optimal gyro_offset_yaw = 61
now I have -8° ... 13° /flight
It  depends .... 
Changing filtering also changes the drift !
It also drifts more on 4S LiPo than on 3S LiPo .